demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1133,12 +1139,46 @@ def get_general_data(path: str, packs: Dict[str, Dict] = None):
     pack = get_pack_name(path)
     marketplaces = get_item_marketplaces(path, item_data=json_data, packs=packs)
 
+    if not id_ and 'marketplacev2' in marketplaces:
+        id_ = f"{pack}-{name}"
+
     data = create_common_entity_data(path=path, name=name, to_version=toversion, from_version=fromversion, pack=pack, marketplaces=marketplaces)
     if brandname:  # for classifiers
         data['name'] = brandname
     return {id_: data}
 
 
+def get_trigger_data(path: str, packs: Dict[str, Dict] = None):
+    json_data = get_json(path)
+    id_ = json_data.get('RULE_ID')
```

Comment:
Notice this was changed
```suggestion
    id_ = json_data.get('trigger_id')
```

Reply:
done